rust/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "spglib"
+version = "0.1.2"
+authors = ["Seaton Ullberg <seatonullberg@gmail.com>"]
+edition = "2018"
+license = "BSD-3-Clause"
+description = "Rust-y bindings to spglib"
+repository = "https://github.com/seatonullberg/spglib"
```

Comment:
Shouldn't this URL be changed to `https://github.com/atztogo/spglib` prior to merge?

Reply:
Good catch, fixed in https://github.com/atztogo/spglib/pull/89/commits/ec957715a6d3800a2a0a4a6bca6a489474b982c1